frontend/src/main/scala/akkaviz/frontend/FrontendApp.scala

```diff
@@ -214,7 +214,7 @@ object FrontendApp extends JSApp with Persistence with PrettyJson with Manipulat
     document.getElementById("receivedelay").appendChild(receiveDelayPanel.render) //FIXME: port to component
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Take the required action to fix the issue indicated by this comment.](https://www.codacy.com/app/lustefaniak/akka-viz/file/2287948829/issues/source?bid=3121282&fileBranchId=3189767#l214)